# In Chromium OS the toolchain has `cros` as a vendor. Such toolchains aren't
# used outside CrOS so LiteX doesn't know these can be used.

--- a/litex/soc/cores/cpu/__init__.py
+++ b/litex/soc/cores/cpu/__init__.py
@@ -56,6 +56,7 @@ CPU_GCC_TRIPLE_RISCV64 = (
     "riscv64-linux-gnu",
     "riscv-sifive-elf",
     "riscv64-none-elf",
+    "riscv64-cros-elf",
 )
 
 CPU_GCC_TRIPLE_RISCV32 = CPU_GCC_TRIPLE_RISCV64 + (
@@ -64,6 +65,7 @@ CPU_GCC_TRIPLE_RISCV32 = CPU_GCC_TRIPLE_RISCV64 + (
     "riscv32-elf",
     "riscv-none-embed",
     "riscv-none-elf",
+    "riscv32-cros-elf",
 )
 
 # CPUS ---------------------------------------------------------------------------------------------
